build/PackageDiffIgnore.xml

```diff
@@ -1010,7 +1010,7 @@
 			<Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.UnregisterPropertyChangedCallback(Microsoft.UI.Xaml.DependencyProperty dp, System.Int64 token)" reason="Not present in WinAppSDK 1.2" />
 			<Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.add_DataContextChanged(Windows.Foundation.TypedEventHandler`2&lt;Microsoft.UI.Xaml.FrameworkElement,Microsoft.UI.Xaml.DataContextChangedEventArgs&gt; value)" reason="Not present in WinAppSDK 1.2" />
 			<Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.remove_DataContextChanged(Windows.Foundation.TypedEventHandler`2&lt;Microsoft.UI.Xaml.FrameworkElement,Microsoft.UI.Xaml.DataContextChangedEventArgs&gt; value)" reason="Not present in WinAppSDK 1.2" />
-			<Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.ClearBindings()" reason="Not present in WinAppSDK 1.2" />
+	        <Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.ClearBindings()" reason="Not present in WinAppSDK 1.2" />
```

Comment:
avoid whitespace changes if possible
```suggestion
		<Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.ClearBindings()" reason="Not present in WinAppSDK 1.2" />
```
remember to rebase squash after, otherwise it is still gonna stain the history